### ISSUE TITLE
jsons can't deserialize NamedTuple

### ISSUE THREAD

**Issue by cypreess** [ramonhagenaars/jsons#15]
When trying to deserialize NamedTuple I got following error:

```
  File "/Users/cypreess/.virtualenvs/.../lib/python3.6/site-packages/jsons/deserializers.py", line 83, in default_tuple_deserializer
    tuple_types = getattr(cls, '__tuple_params__', cls.__args__)
AttributeError: type object 'MyNamedTuple' has no attribute '__args__'
```

This is different error than related issue which was solved: https://github.com/ramonhagenaars/jsons/issues/11

**Comment by ramonhagenaars** [comment]
Thanks cypreess, for another issue.
I consider this a bug.

**Comment by ramonhagenaars** [comment]
Fixed in [0.7.1](https://pypi.org/project/jsons/)
